### ISSUE TITLE
A11y_SQLAzureDataStudio_ConnectToASQLServer_ConnectionDetails_ScreenReader: Table mark-up announced unnecessarily below connection details tab.

### ISSUE THREAD

**Issue by v-atkulk** [microsoft/azuredatastudio#9185]
[“Check out Accessibility Insights! ](https://accessibilityinsights.io/)- Identify accessibility bugs before check-in and make bug fixing faster and easier.”

#A11yMAS;#A11yTCS;#SQL Azure Data Studio;#DesktopApp;#Win32;#MAS1.3.1;#NVDA;#Benchmark;#A11y_SQLAzureDataStudio;#A11ySev2; #JAWS;
 
**Environment Details:**
Version: 1.16.0-insider
Commit: 506c6a5e5f8423d3249670f55b1466b1980a5f03
Date: 2020-02-17T05:43:18.399Z
VS Code: 1.42.0
Electron: 7.1.11
Chrome: 78.0.3904.130
Node.js: 12.8.1
V8: 7.8.279.23-electron.0
OS: Windows_NT x64 10.0.18363
 
**Additional Details:**
Screen reader: NVDA(2019.3.1), JAWS(2020.2001.70)
MAS reference: MAS1.3.1
 
**Repro Steps:**
1. Launch Azure Data Studio.
2. Select the Connections icon (if not already selected) from left bar.
3. Hover over the 'Servers' header and select the 'New Connection' icon.
4. Navigate to connection details.
5. Launch NVDA and navigate inside connection details screen.
6. Check if all announcements are made properly by screen reader.
 
**Actual:**
After navigating to the connection details screen, unnecessary table markup is being announced/assigned to the label and input fields present on the page.

**Same issue observed with JAWS also.**

**Expected:**
While navigating under the connection details screen no table markup should be announced for labels and input fields unless they are visually identified as a table.

**User Impact:**
Users might get confused as there is no table present yet the table markup is assigned to all the labels and input fields.

**Attachment link reference:**
[9185_A11y_SQLAzureDataStudio_Connect to a SQL Server_Connection Details_ScreenReader.zip](https://github.com/microsoft/azuredatastudio/files/4291575/9185_A11y_SQLAzureDataStudio_Connect.to.a.SQL.Server_Connection.Details_ScreenReader.zip)

**Comment by alanrenmsft** [comment]
@v-atkulk please fix the attachment link, i am not able to access it and i am not able to figure out what is wrong by reading the description.

**Comment by alanrenmsft** [comment]
please reactivate the issue after fixing the attachment. i am closing the issue as it is not actionable for me.

**Comment by Harika1702** [comment]
<img width="960" alt="9185_A11y_Fixed" src="https://user-images.githubusercontent.com/100270074/218719120-47736ed9-4339-45e1-bb58-64e5c66b16f9.png">
Now issue is not reproducing.
GitHub Tags:#Closed;

**Comment by mstechie** [comment]
GitHubTags:#SQLADS-Win32-Feb2020;
